lib/WTSI/NPG/HTS/PacBio/Sequel/AnalysisPublisher.pm

```diff
@@ -186,9 +187,23 @@ sub publish_sequence_files {
          ($format eq $SEQUENCE_FASTA_FORMAT))
           ? 0 : 1;
 
+      my $tags;
+      if ($is_target) {
+        $tags = npg_warehouse::loader::pacbio::product->get_tags($records[0]);
```

Comment:
`npg_warehouse::loader::pacbio::product` is a role, so calling a function in its namespace like this is an unexpected use of it. The role should be consumed by a class that can be instantiated and the `get_tags` called as an instance method.